neuromation/api/config_factory.py

```diff
@@ -63,11 +64,15 @@ def __init__(
         self,
         path: Optional[Path] = None,
         trace_configs: Optional[List[aiohttp.TraceConfig]] = None,
+        trace_id: Optional[str] = None,
     ) -> None:
         if path is None:
             path = Path(os.environ.get(CONFIG_ENV_NAME, DEFAULT_CONFIG_PATH))
         self._path = path.expanduser()
-        self._trace_configs = trace_configs
+        self._trace_configs = [_make_trace_config()]
+        if trace_configs:
+            self._trace_configs += list(trace_configs)
```

Comment:
Explicit converting to list is not needed.

Reply:
thanks